### DISCUSSION TITLE
How can I have Label concatenation

### DISCUSSION THREAD

**Discussion by balduinofernando** [sagalbot/vue-select#1263] [Q&A] [answered]
How can I concatenate the label for my vue-select component?

Let's say I want to concatenate the name of the VAT description with it's tax amount.

How can I do it?

**Comment by wazum** [comment, marked as answer]
Use the slots, see [documentation](https://vue-select.org/api/slots.html), example:

```
<template slot="option" slot-scope="option">
    {{ option.label }}<br>
    <small class="text-muted">
        {{ option.address.zipCode }}
        {{ option.address.city }},
        {{ fullStateName(option.address.country, option.address.state) }},
        {{ fullCountryName(option.address.country) }}
    </small>
</template>
```

**Comment by titosobabas** [comment]
> Use the slots, see [documentation](https://vue-select.org/api/slots.html), example:
> 
> ```
> <template slot="option" slot-scope="option">
>     {{ option.label }}<br>
>     <small class="text-muted">
>         {{ option.address.zipCode }}
>         {{ option.address.city }},
>         {{ fullStateName(option.address.country, option.address.state) }},
>         {{ fullCountryName(option.address.country) }}
>     </small>
> </template>
> ```

Doesn't worked.

**Comment by wazum** [comment]
@titosobabas  I use this in production, so rest assured it works ;-) post some code on how your option looks like …
